Check `layered_dag` and `dag` (#8281)
sorbet/sorbet: website/docs/error-reference.md

  `--packager-layers util,lib,app`, all imports for a package with layer `lib`
  must either also have layer `lib`, or have layer `util` (but not layer `app`).

## 3727

> This error is specific to Stripe's custom `--stripe-packages` mode. If you are
> at Stripe, please see [go/modularity](http://go/modularity) and
> [go/strict-dependencies](http://go/strict-dependencies) for more.

> If a package is at `strict_dependencies 'layered_dag'` or stricter, it can not

Reviewer: ```suggestion
If a package is at `strict_dependencies 'layered_dag'` or stricter, it cannot
```
